airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -270,6 +275,7 @@ def __init__(
         reattach_on_restart: bool = True,
         startup_timeout_seconds: int = 120,
         get_logs: bool = True,
+        container_logs: list[str] | str | Literal[True] = BASE_CONTAINER_NAME,
```

Comment:
```suggestion
        container_logs: Iterable[str] | str | Literal[True] = BASE_CONTAINER_NAME,
```

Reply:
NIT. but that would be consistent with the code.

Reply:
Sure, makes sense. Doesn't make sense to have list type when we say sequence. Good catch!